.github/workflows/scripts/ping-codeowners-on-new-issue.sh

```diff
@@ -68,6 +70,7 @@ for COMPONENT in ${BODY_COMPONENTS}; do
   fi
 done
 
+# TODO: This check isn't working, it always returns no related components
```

Comment:
I don't understand, should this code be removed or is there a follow-up?

Reply:
This should be investigated in the future. From my local testing I couldn't get this check to fail, so it's more of an open question whether or not the check is useful or should be removed.